portal/src/main/webapp/js/lib/EnhancedFixedDatatable.js

```diff
@@ -144,10 +144,19 @@ var QtipWrapper = React.createClass({displayName: "QtipWrapper",
         }
 
         if (attr === 'CASE_ID') {
-            label = React.createElement("a", {target: "_blank", href: cbio.util.getLinkToSampleView(cancerStudyId, label)}, label)
+            label = React.createElement("a", {target: "_blank", href: (window.hasOwnProperty('cbio') && cbio.hasOwnProperty('util'))?cbio.util.getLinkToSampleView(cancerStudyId, this.props.rawLabel):''}, label)
```

Comment:
Yes. This is the library copy. In the library itself, it doesn't have the cbio.util